### PR TITLE
Minor updates to 2022-12-13-host-process-containers-ga blog

### PR DESCRIPTION
**Pull request by marosset**
Signed-off-by: Mark Rossetti <marosset@microsoft.com>

Here are some minor updates to the HostProcess Containers blog-post suggested in https://github.com/kubernetes/website/pull/37370

/assign @sftim @fsmunoz

### CODE REVIEW

File: content/en/blog/_posts/2022-12-13-host-process-containers-ga/index.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ containers, has finally made it to **GA in Kubernetes 1.26**!
 
 What are HostProcess containers and why are they useful?
 
-Cluster operators are often faced with the need to configure their nodes upon provisioning. Whether it's
+Cluster operators are often faced with the need to configure their nodes upon provisioning. Such as
```
Comment (jsturtevant) [inline content/en/blog/_posts/2022-12-13-host-process-containers-ga/index.md:15]:
```suggestion
Cluster operators are often faced with the need to configure their nodes upon provisioning such as
```

I think this is correct?

**Review by sftim — COMMENTED** [review]
If you're willing to, I wonder about also changing the title to reflect that the release has happened.

Eg: `title: "Kubernetes 1.26: Windows HostProcess Containers Are Generally Available"`


Anyway,
/lgtm
/approve

/hold
OK to unhold if you're happy with it as it stands.

**Comment by marosset** [comment]
> If you're willing to, I wonder about also changing the title to reflect that the release has happened.
> 
> Eg: `title: "Kubernetes 1.26: Windows HostProcess Containers Are Generally Available"`
> 
> Anyway, /lgtm /approve
> 
> /hold OK to unhold if you're happy with it as it stands.

Updated

**Pull request merged**